src/Sentry.Maui.CommunityToolkit.Mvvm/Sentry.Maui.CommunityToolkit.Mvvm.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>MAUI and Community Toolkit integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
+    <TargetFrameworks>net9.0;net8.0</TargetFrameworks>
+
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <UseMaui>true</UseMaui>
+
+    <!-- Skip warnings about including Microsoft.Maui.Controls... that should be done from the application -->
+    <SkipValidateMauiImplicitPackageReferences>true</SkipValidateMauiImplicitPackageReferences>
+    <RootNamespace>Sentry.Maui.CommunityToolkit.Mvvm</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommunityToolkit.Mvvm" Version="8.4.0"/>
```

Comment:
Do we need to use the latest and greatest [version of this package](https://www.nuget.org/packages/CommunityToolkit.Mvvm#versions-body-tab)? Ideally we'd depend on the oldest version of this package that works functionally... that way SDK users don't have to upgrade this package to use Sentry (but they could use the latest and greatest as well if they wanted).

Reply:
I wanted to make sure I didn't have a range of versions to deal with.  I do this with open source as well.  If people have a need for lesser versions, it can be obviously be discussed, but we have exactly 1 feature requests for this right now - me :)

Reply:
That optimises for us as the maintainers (it makes our job really easy if we only have one potential version to deal with).

As a general principal, as SDK authors we try to target the oldest versions of things that we possibly can. The reason being that that generally makes it easier for our users. If we give them the greatest possible set of options about what versions of dependencies they need, we constrain them less in the complex system that they're building.

If this PR is genuinely just for an audience of 1, let's not waste any more time on it. Assuming it's for a wider audience than that though, the principal above as served us well in the past.